spec/directions/bake_example_spec.rb

```diff
@@ -153,6 +153,97 @@
     end
   end
 
+  context 'when there is more than one exercise with solutions' do
+    let(:exercise) do
```

Comment:
I'm sorry if I'm missing the obvious but I don't see where `exercise` is being used?

Reply:
it fits into :example above (so the example wrapper stays the same, but the content parts can get swapped around)